Contracts/Contracts.csproj

```diff
@@ -21,6 +21,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\IO\CommunAxiom.Commons.Ingestion.csproj" />

```

Comment:
This reference does not belong there. The Ingestion project should be pointing to Contracts for its own contracts.